src/languages.rs

```diff
@@ -385,7 +385,7 @@ impl Iso639a {
     /// * `lang` - Input string to validate (case-insensitive)
     ///
     /// # Examples
-    /// ```
+    /// ```ignore
```

Comment:
This is just a comment, so in the `actix_error_proc` the ignore tag brought a lot of warnings to the documentation, so we need to check into that in a future.